test/e2e/framework/util.go

```diff
@@ -74,6 +74,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/master/ports"
 	taintutils "k8s.io/kubernetes/pkg/util/taints"
+	e2ecloudgcp "k8s.io/kubernetes/test/e2e/cloud/gcp"
```

Comment:
Please don't import outside of e2e framework in k/k from e2e framework.
That is an invalid dependency.